### PR TITLE
feat(MOPLAT-237): update indicator and label styles

### PR DESCRIPTION
**Pull request by patrinoua**
This PR resolves [MOPLAT-237] 

### Description

updates indicator and label styles

### PR Checklist

- [x] I tested my changes on **iOS**
- [ ] I tested my changes on **Android**.
- [x] I added screenshots or videos to illustrate my changes.
## Todo 
- [ ] add Tests: There is [another PR](https://github.com/artsy/energy/pull/15) that introduces the configuration for tests, once this is done we can add tests for the HomeTab as well. Ticket https://artsyproduct.atlassian.net/browse/MOPLAT-241

### To the reviewers 👀

- [x] Please some of the reviewers also test this PR on the android simulator or device.

https://user-images.githubusercontent.com/36475005/167438149-1de8388b-a4f9-4936-acaa-eff62d86d4c9.mov

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look on our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-237]: https://artsyproduct.atlassian.net/browse/MOPLAT-237?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by rajsam003** [comment]
Great work 🕵🏻‍♀️
I just added a few comments.

**Comment by rajsam003** [comment]
@patrinoua Any update on this, please?

File: src/Scenes/HomeTabs/HomeTabs.tsx [hunk 1]
```diff
@@ -33,14 +33,12 @@ export const HomeTabs = () => {
...
           scrollEnabled
-          indicatorStyle={{
-            backgroundColor: "black",
-            maxWidth: 40,
-            height: 1,
-            marginHorizontal: 14,
-          }}
+          {...props}
+          style={{ marginHorizontal: 10 }}
+          labelStyle={{ margin: -10 }}
```
[the remark this answers is not in the document]
Comment (patrinoua) [reply src/Scenes/HomeTabs/HomeTabs.tsx]: We also spent some time investigating this with @admbtlr earlier and saw that there is no good way of changing this without creating a patch to the package, and therefore adding a negative margin seems like a preferable solution.

File: src/Scenes/HomeTabs/HomeTabs.tsx [hunk 2]
```diff
@@ -33,14 +33,12 @@ export const HomeTabs = () => {
       containerStyle={{ paddingTop: 20 }}
       TabBarComponent={(props) => (
         <MaterialTabBar
-          {...props}
           scrollEnabled
-          indicatorStyle={{
-            backgroundColor: "black",
-            maxWidth: 40,
-            height: 1,
```
[the remark this answers is not in the document]
Comment (patrinoua) [reply src/Scenes/HomeTabs/HomeTabs.tsx]: done

**Pull request merged**